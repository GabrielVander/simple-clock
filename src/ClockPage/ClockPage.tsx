import React from "react";
import Clock from "./Clock/Clock";
import Navbar from "react-bootstrap/Navbar";

function ClockPage() {
    return (
        <div className="App">
            <div className="App-header">
                <Navbar
                    bg="transparent"
                    fixed="top"
                    className='justify-content-center'
                >
                    <Navbar.Text style={{ color: 'white' }} >
                        Simple Clock
                    </Navbar.Text>
                </Navbar>
                <Clock/>
            </div>
        </div>
    );
}

export default ClockPage;
