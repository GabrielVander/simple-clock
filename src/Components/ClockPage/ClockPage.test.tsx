import React from "react";
import {render} from "@testing-library/react";
import ClockPage from "./ClockPage";

test('Renders correctly', () => {
    const component = render(<ClockPage/>);
    expect(component).toMatchSnapshot();
});