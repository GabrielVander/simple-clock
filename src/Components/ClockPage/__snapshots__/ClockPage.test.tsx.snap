// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="App-header"
        >
          <nav
            class="justify-content-center navbar navbar-expand navbar-light bg-transparent fixed-top"
          >
            <span
              class="navbar-text"
              style="color: white;"
            >
              Simple Clock
            </span>
          </nav>
          <div
            class="container"
          >
            <div
              class="row"
            >
              <div
                class="col"
              >
                <div>
                  <div
                    class="fade alert alert-danger alert-dismissible show"
                    role="alert"
                  >
                    <button
                      class="close"
                      type="button"
                    >
                      <span
                        aria-hidden="true"
                      >
                        ×
                      </span>
                      <span
                        class="sr-only"
                      >
                        Close alert
                      </span>
                    </button>
                    <p>
                      Failed to load illustration
                    </p>
                  </div>
                </div>
                <div
                  class="custom-container container"
                >
                  <div
                    class="row"
                  >
                    <div
                      class="col"
                    >
                      <div
                        class="card-container card"
                      >
                        <div
                          class="card-body"
                        >
                          <div
                            class="row no-gutters"
                          >
                            <div
                              class="col"
                            >
                              17
                            </div>
                            <div
                              class="border-right border-left col"
                            >
                              45
                            </div>
                            <div
                              class="border-right border-left col"
                            >
                              53
                            </div>
                            <div
                              class="col"
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="App-header"
      >
        <nav
          class="justify-content-center navbar navbar-expand navbar-light bg-transparent fixed-top"
        >
          <span
            class="navbar-text"
            style="color: white;"
          >
            Simple Clock
          </span>
        </nav>
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div>
                <div
                  class="fade alert alert-danger alert-dismissible show"
                  role="alert"
                >
                  <button
                    class="close"
                    type="button"
                  >
                    <span
                      aria-hidden="true"
                    >
                      ×
                    </span>
                    <span
                      class="sr-only"
                    >
                      Close alert
                    </span>
                  </button>
                  <p>
                    Failed to load illustration
                  </p>
                </div>
              </div>
              <div
                class="custom-container container"
              >
                <div
                  class="row"
                >
                  <div
                    class="col"
                  >
                    <div
                      class="card-container card"
                    >
                      <div
                        class="card-body"
                      >
                        <div
                          class="row no-gutters"
                        >
                          <div
                            class="col"
                          >
                            17
                          </div>
                          <div
                            class="border-right border-left col"
                          >
                            45
                          </div>
                          <div
                            class="border-right border-left col"
                          >
                            53
                          </div>
                          <div
                            class="col"
                          />
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
